conda_spawn/shell.py

```diff
@@ -164,6 +164,41 @@ def executable(self):
         return "zsh"
 
 
+class ShellShell(PosixShell):
```

Comment:
I think we should rename this to avoid confusion:

```suggestion
class PrefixShellShell(PosixShell):
```